py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -17,11 +17,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-
+http_exception = None
 try:
     import http.client as http_client
 except ImportError:
     import httplib as http_client
+    http_exception = http_client.HTTPException
```

Comment:
I am a pythonista :)

I would keep the imports as they were:
```python
try:
    import http.client as http_client
except ImportError:
    import httplib as http_client
```

then just use it as `http_client.HTTPException`, like:

```python
except (http_client.BadStatusLine, socket.error, http_client.HTTPException):
```

This way is idiomatic, and follows the code conventions (see: `http_client.BadStatusLine`).

Please remove the import/namespace trickery from this PR.